cypress/e2e/5-ndv.cy.ts

```diff
@@ -163,5 +163,70 @@ describe('NDV', () => {
 			ndv.getters.outputPanel().find('[class*=_pagination]').should('not.exist');
 			ndv.getters.outputPanel().find('[data-test-id=run-data-schema-item] [data-test-id=run-data-schema-item]').should('have.length', 20);
 		})
-	})
+	});
+
+	it('can link and unlink run selectors between input and output', () => {
+		cy.fixture('Test_workflow_5.json').then((data) => {
```

Comment:
There's a command for this which also takes care about waiting for the WF to load `cy.createFixtureWorkflow('createFixtureWorkflow')`

Reply:
ah just copy paste.. will update